python/nano/src/bigdl/nano/quantization/__init__.py

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from .quantization_inc import QuantizationINC
```

Comment:
why import here?

Reply:
import here, we can import `QuantizationINC` by:
```python
from bigdl.nano.quantization import QuantizationINC
```
otherwise:
```python
from bigdl.nano.quantization.quantization_inc import QuantizationINC
```

Reply:
I think user does not need to use `QuantizationINC` directly? Still I have no issue to import it here.